templates/base.html

```diff
@@ -0,0 +1,49 @@
+{% load static %}
+<html>
+	<head>
+		<meta charset="utf-8">
+		{% block content %}
+		{% endblock %}
+	</head>
+	<body>
+		<!--Header Block-->
+		<header class="header shift-center">
+			{% block content %}
```

Comment:
I'm not sure if you can reuse block names. I've always used names like head, content, footer for placing main content.

Reply:
I am also not sure I just used it. Anyways now I have made all the names different.